### PR TITLE
Story AB#1050411: concurrentqueue.h(407): warning C4554: '<<': check operator precedence for possible error; use parentheses to clarify precedence

### PR DESCRIPTION
**Pull request by v-lomer**
Issue flagged with VS2022 v143 toolchain

### CODE REVIEW

File: concurrentqueue.h [hunk 1]
```diff
@@ -1,3640 +1,3633 @@
...
-private:
-#endif
-	
-	///////////////////////////
-	// Producer base
-	///////////////////////////
-
-	//MSVC in VS 15.7 Seems to do template expantion in a different order, these forward declares fix
-	//     that MSVC can't find definitions for these two structs in dequeue and dequeue_bulk
+// Provides a C++11 implementation of a multi-producer, multi-consumer lock-free queue.
```
Comment (dango-msft) [inline concurrentqueue.h:1]: Do you know why the ENTIRE file show as changed? Is that because the file is too big for GitHub to handle?

Comment (v-lomer) [reply concurrentqueue.h:1]: I will look into this some more. Only one line was changed locally.

Comment (v-lomer) [reply concurrentqueue.h:1]: Using https://github.com/Mojang/concurrentqueue/pull/5 instead

**Comment by v-lomer** [comment]
Using https://github.com/Mojang/concurrentqueue/pull/5 as the file was 100% `replaced` in this PR to cannot see the couple of changes made.

**Pull request closed without merging**